templates/template/readonly.html.twig

```diff
@@ -0,0 +1,10 @@
+{% extends "template/UI.html.twig" %}
+
+{% block title %}
+    System is in read-only mode
```

Comment:
This isn't really read-only though. We should probably ctall this maintenance mode across the board instead, especially since there's a separate ticket in the system for a true read-only mode.